vendor/github.com/openshift/api/config/v1/0000_10_config-operator_01_infrastructure.crd.yaml

```diff
@@ -220,6 +220,13 @@ spec:
                     - HighlyAvailable
                     - SingleReplica
                     - External
+                cpuPartitioning:
+                  description: cpuPartitioning expresses that CPU partitioning is a currently enabled feature in the cluster.
+                  type: string
```

Comment:
should there be pattern recognition to filter out bogus options at API validation step? though, the pattern may be too copmlicated because there are a variable number of `,` and ranges allowed..

Reply:
That's a good point, I think David might have had the same input in the API PR for this, is this what you had in mind https://github.com/openshift/api/pull/1284#discussion_r971268312

Reply:
not quite, I was talking about validating that the cpuset is in a format we epxect like `0,2-4,7`  but come to think of it, the validation would not be trivial, so I am fine without.